Tools/Schema.NET.Tool/Schema.NET.Tool.csproj

```diff
@@ -1,62 +1,42 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup Label="Build">
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net8.0;net6.0;netstandard2.0</TargetFrameworks>
```

Comment:
Source generators can only target `netstandard2.0` unfortunately.

Reply:
I can even remove netstandard2.0 completely from the targets and it continues to function no problem.

Reply:
I will look into it.

Reply:
I split Tool project into separate projects to allow proper package versioning for different targets when built from console (and published as a package) - as you cannot properly reference different versions of System.Text.Json and others. While allowing usage from Visual Studio and falling back to netstandard2.0 only.

Reply:
I fixed build for Visual Studio, Rider and CLI